File: ajax/jsonp.php
<?php

header('Content-type: application/javascript');

$data['ip'] = $_SERVER['REMOTE_ADDR'];
$data['host'] = $_SERVER['REMOTE_HOST'];
$data['ref'] = $_SERVER['HTTP_REFERER'];
$data['lang'] = $_SERVER['HTTP_ACCEPT_LANGUAGE'];

echo json_encode($data);

?>

File: ajax/jsonp.pl
#!/usr/bin/perl

use strict;
use warnings;

print "Content-type: application/javascript\n\n";

print "{\"ip\":\"".$ENV{'REMOTE_ADDR'}."\",".
        "\"ref\":\"".$ENV{'HTTP_REFERER'}."\",".
        "\"lang\":\"".$ENV{'HTTP_ACCEPT_LANGUAGE'}."\"}";

1;